src/Application/Common/Exceptions/LoginFailedException.cs

```diff
@@ -14,7 +14,7 @@ public LoginFailedException(string message)
 
     // TODO: write unit tests using multiple reasons as input
     public LoginFailedException(string[] reasons)
-        : base("Login failed: " + string.Join(", ", reasons))
```

Comment:
I've removed this as I would like to return to the UI the exception details directly, and I thought displaying `Login failed:` before the message was redundant. If you think otherwise please let me know